tokio/src/sync/broadcast.rs

```diff
@@ -949,7 +949,7 @@ impl<T: Clone> Receiver<T> {
     /// # Examples
     ///
     /// ```
-    /// use tokio::stream::StreamExt;
+    /// use tokio_stream::StreamExt;
     /// use tokio::sync::broadcast;
     ///
```

Comment:
This doc comment is on a method called `into_stream` which returns `impl Stream`.